src/devices/HardwareMonitor/HardwareMonitor.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="OpenHardwareMonitor.cs" />
+    <Compile Include="SensorType.cs" />
+    <None Include="README.md" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Management" Version="5.0.0-preview.5.20224.13" />
```

Comment:
See my other comment, basically this versio should be comming from eng\Versions.props